### PR TITLE
Automated srf gen

### PR DESCRIPTION
**Pull request by jpa198**
#### Checklist
- Have you updated the README (yes/no)?
- Have you updated the CHANGELOG (yes/no)?

### CODE REVIEW

**Comment by jpa198** [comment]
The one failing test depends on the parallel qcore PR

**Review by sungeunbae — COMMENTED** [review]
Considering the significance and size of this PR, it deserves some description.

File: workflow/automation/install_scripts/install_realisation.py [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+    parser.add_argument("--log_dir", type=Path)
+
+    vm_pert = parser.add_mutually_exclusive_group()
+    vm_pert.add_argument(
+        "--vm_perturbations",
+        action="store_true",
+        help="Use velocity model perturbations. If this is selected all realisations must have a perturbation file.",
+    )
+    vm_pert.add_argument(
+        "--ignore_vm_perturbations",
```
Comment (sungeunbae) [inline workflow/automation/install_scripts/install_realisation.py:20]: Option name is slightly long, but I can accept this

Comment (sungeunbae) [reply workflow/automation/install_scripts/install_realisation.py:20]: What happens both --vm_perturbations and --ignore_vm_perturnations are set? I can sense these two are not meant to be behave in opposite ways...or are they?

**Review by sungeunbae — COMMENTED** [review]
Just have a comment in relation to --vm_perturbations argument. Other than that, can't fault anything. Well done.

File: workflow/automation/install_scripts/install_realisation.py (hunk 1 above)
Comment (jpa198) [reply workflow/automation/install_scripts/install_realisation.py:20]: On line 13 a mutually exclusive group is made, so argparse will prevent both options being chosen at the same time
The option name is inherited from the current install_cybershake.py script. But in this case, the script is not expected to be used by a user, only through the automated workflow so length of argument name is less important than clarity

**Review by joelridden — APPROVED** [review]

**Review by sungeunbae — APPROVED** [review]

**Pull request merged**